### DISCUSSION TITLE
10月份起，就登录不了专业版了，也给你们邮件反馈了账单和账号，到现在都没有反馈，你们还在正常运营吗》

### DISCUSSION THREAD

**Discussion by Sanwan777** [getlantern/lantern-forum-cn#1082] [Q&A]
10月份起，就登录不了专业版了，也给你们邮件反馈了账单和账号，到现在都没有反馈，你们还在正常运营吗》

**Comment by MaxMKMax** [comment]
@Sanwan777 您好. 估计客服哪里漏掉了. 您在这里给我发账号邮箱和支付单据. 我帮您查看.

**Comment by MaxMKMax** [comment]
@Sanwan777 我这里会帮您追问好的.

**Comment by MaxMKMax** [comment]
@Sanwan777 您好. 请您在这里给我发您的账号邮箱和支付单据. 我帮您跟进一下.

**Comment by Sanwan777** [comment]
***@***.***，难道你们不能根据我的账号查询么



---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年12月17日 23:41 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 10月份起，就登录不了专业版了，也给你们邮件反馈了账单和账号，到现在都没有反馈，你们还在正常运营吗》 (Discussion #1082) |

@Sanwan777 您好. 请您在这里给我发您的账号邮箱和支付单据. 我帮您跟进一下.
